pulp_rpm/tests/functional/utils.py

```diff
@@ -250,3 +250,10 @@ def init_signed_repo_configuration():
          "AsciiArmoredDetachedSigningService.objects.create(name='sign-metadata',"
          f"script='{signing_script_path}')")
     )
+
+
+def get_package_repo_path(package):
+    """Get package repo path with directory structure."""
+    return os.path.join(
+        "Packages", package.lower()[0], package
```

Comment:
Does it make sense to have `"Packages"`  as a constant? It's used at least 3 times in this PR.